commerce_coordinator/apps/titan/views.py

```diff
@@ -2,6 +2,96 @@
 Views for the titan app
 """
 
-# from django.shortcuts import render
+import logging
 
-# Create your views here.
+from edx_rest_framework_extensions.permissions import LoginRedirectIfUnauthenticated
+from rest_framework.authentication import BasicAuthentication, SessionAuthentication
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.response import Response
+from rest_framework.throttling import UserRateThrottle
+from rest_framework.views import APIView
+
+from commerce_coordinator.apps.core.signal_helpers import format_signal_results
+
+from .serializers import OrderFulfillSerializer
+from .signals import fulfill_order_placed_signal
+
+logger = logging.getLogger(__name__)
+
+
+class OrderFulfillView(APIView):
+    """
+    API for order fulfillment that is called from Titan.
+    """
+    permission_classes = [LoginRedirectIfUnauthenticated]
+    authentication_classes = [SessionAuthentication, BasicAuthentication]
```

Comment:
I think we can remove `authentication_classes`. It's defined upstream.

Reply:
Gotcha, will remove from LMS app views as well